src/templates/admin/edit.pug

```diff
@@ -0,0 +1,57 @@
+extends ../components/base.pug
+
+block title
+  title #{title} : #{verb} Admin #{admin ? admin.name : ''}
+
+block content
+  div.columns
+    div.column
+      h3.title.is-3.has-icon
+        span.icon.is-medium
+          i.fas.fa-edit
+        span #{verb} Admin #{admin ? admin.name : ''}
+      div
+        form(method='post' action='/admins/edit')
+          if admin
+            input(type='hidden' name='steam_id' value=admin.steam_id)
+          else
+            div.field
+              label.label(for='admin.steam_id') Steam ID:
+              p.control
+              input.input(id='steam_id' type='text' name='steam_id' placeholder='12345678' required)
+          if admin
+            div.field
+              label.label(for='admin.division_id') Division:
+              p.control
+                select.form-control.input-lg(name='division_id')
+                  option(value='0', selected= true) -- Select --
```

Comment:
Instead of `'0'` in these cases, let's use an empty string. It might also make sense to `disabled` the "-- Select --" option and create a specific "All Divisions" or similar option for each.